umm-spec-lib/src/cmr/umm_spec/test/iso_smap_expected_conversion.clj

```diff
@@ -232,17 +247,50 @@
          (map cmn/map->DataCenterType)
          distinct)))
 
+(defn- not-provided-begin-date
+  "Returns the default java epoch zero used for not provided date values.
+   This function returns the def record and not just a map so that
+   it can be compared to the actual round trip translation."
+  []
+  (for [x (range 0 1)]
```

Comment:
Thanks for helping me here too.  I had spacing issues with using the brackets.  I have updated the code.